### DISCUSSION TITLE
[ Feature Request ] Caller ID Support?

### DISCUSSION THREAD

**Discussion by OrhanYigitDurmaz** [Blair2004/NexoPOS#1938] [Ideas]
Hi. I would like to use this in one of my stores. Currently a worker sits in front of the computer and takes orders with a landline phone, which is connected to a Caller ID. Right now when someone calls, a popup shows up showing customers details, what they buy most often, their last buy date and a take order button. I would like to do this in NexoPOS or something similar. The Caller ID uses a .dll file to communicate with our software, which i can use with python. How should i go about this? SHould this be a module? How should i trigger popups?

**Comment by Blair2004** [comment]
Hi,
First of all Sorry for th delay.
Now, NexoPOS is mostly hosted online, it can't have access to .dll file installed locally.
This approach is therefore not a great fit unfortunately.
